src/photos.cs

```diff
@@ -702,11 +722,18 @@ interface PHContentEditingOutput : NSCoding, NSSecureCoding {
 
 		[Export ("renderedContentURL", ArgumentSemantic.Copy)]
 		NSUrl RenderedContentUrl { get; }
+
+#if false
```

Comment:
upon verification it was added in the wrong type - that's what happen when you're half blind :|